wiki/Client/Beatmap_editor/Menu/de.md

```diff
@@ -2,242 +2,242 @@
-| Skalieren... (`Strg` + `Shift` + `S`) | Veränder den Abstand für die Hit Objekt um ein Vielfaches (0.500x - 1.200x). Sie können die Position festlegen und ob sich die Objekte nur auf einer oder beiden Achsen verschieben werden sollen. |
+| Auswahl umkehren (`Strg` + `G`) | **Slider:** Vertauscht Slideranfang und Sliderende. **Wähle mehrere Objekte aus:** Dein Beatmap-Design wird andersherum ausgerichtet. Z. B. wird ein Hit-Circle bei 10 Sekunden zu den letzten 10 Sekunden eines Songs verschoben und umgekehrt. |
+| Horizontal spiegeln (`Strg` + `H`) | Wendet das/die Hit-Objekt(e) von links nach rechts bzw. von rechts nach links (y-Achse ist die Mittelachse). |
+| Vertikal spiegeln (`Strg` + `J`) | Wendet das/die Hit-Objekt(e) von unten nach oben bzw. von oben nach unten (x-Achse ist die Mittelachse). |
+| 90° im Uhrzeigersinn drehen (`Strg` + `>`) | Hit-Objekt(e) um 90° nach rechts drehen. |
+| 90° gegen den Uhrzeigersinn drehen (`Strg` + `<`) | Hit-Objekt(e) um 90° nach links drehen. |
+| Rotieren ... (`Strg` + `Shift` + `R`) | Lege den Rotationswinkel (+/- 180°) für die Hit-Objekte fest. Du kannst auch die Richtung \[im/gegen Uhrzeigersinn\] und Position \[Mitte im Spielfeld/in der Auswahl\] festlegen. |
+| Skalieren... (`Strg` + `Shift` + `S`) | Verändere die Größe der Hit-Objekte um einen gewissen Faktor (0.500x - 1.200x). Du kannst die Position \[Mitte im Spielfeld/in der Auswahl\] festlegen und ob die Objekte nur auf einer oder beiden Achsen skaliert werden sollen. |
 
 ### Tasten zum Zurücksetzen
 
 | Name | Beschreibung |
 | :-- | :-- |
-| Samples des ausgewählten Objekts zurücksetzen | Entfertn den Hitsound vom ausgewählten Hit Objekt. |
-| Alle Samples zurücksetzen | Entfernt alle festgelegten Hitsounds von den Hit Objekten. |
-| Kombo-Farben zurücksetzen | Setzt alle Kombofarben zurück. |
-| Pausen zurücksetzen | Setzt alle Pausen zurück. Funktioniert nicht bei automatisch platzierten Pausen. |
+| Samples des ausgewählten Objekts zurücksetzen | Entfernt den Hitsound vom ausgewählten Hit-Objekt. |
+| Alle Samples zurücksetzen | Entfernt alle festgelegten Hitsounds von den Hit-Objekten in dieser Schwierigkeitsstufe. |
+| Combo-Farben zurücksetzen | Setzt alle Combo-Farben zurück. |
+| Pausen zurücksetzen | Setzt alle Pausen zurück. Funktioniert nicht bei automatisch platzierten Pausen, die übermäßigen HP-Drain verhindern. |
 
-### Kommandos zum Stubsen
+### Befehle zum Stubsen
 
 | Name | Beschreibung |
 | :-- | :-- |
-| Stubs zurück (`J`) | Bewegt Hit Objekt(e) um ein Beat zurück im Beat Snap Divisor. |
-| Stubs vrfwärts (`K`) | Bewegt Hit Objekt(e) um ein Beat nach vorne im Beat Snap Divisor. |
+| Stubs zurück (`J`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat zurück. |
+| Stubs vorwärts (`K`) | Bewegt Hit Objekt(e) abhängig vom Taktteiler um einen Beat nach vorne. |
 
 ## Ansicht
 
-![Ansichtmenü](img/M_View-DE.jpg "Ansichtmenü")
+![Ansichtsmenü](img/M_View-DE.jpg "Ansichtsmenü")
+
+Interessante Dinge.
```

Comment:
This sounds too casual/incomprehensible when reading this again. In general, some phrases in the original version seem badly written, but this is not of concern here now. Despite this, I suggest to either remove the phrase entirely or change it to `Sehenswürdigkeiten`. If someone has a better idea how to translate just `places of interest`, please make a suggestion!

Reply:
The annoying part about removing lines is that it makes it harder to compare original and translation. Honestly I think it's best as is, but I would rather see it changed to `Sehenswürdigkeiten` than removed entirely.